domains/certificates/Query.API/API/Query.API/Controllers/ContractsController.cs

```diff
@@ -45,21 +48,70 @@ public class ContractsController : ControllerBase
             return ValidationProblem(ModelState);
         }
 
-        var startDate = DateTimeOffset.FromUnixTimeSeconds(createContract.StartDate);
-        DateTimeOffset? endDate = createContract.EndDate.HasValue ? DateTimeOffset.FromUnixTimeSeconds(createContract.EndDate.Value) : null;
+        var startDate = UnixTimestamp.Create(createContract.StartDate);
+        UnixTimestamp? endDate = createContract.EndDate.HasValue
+            ? UnixTimestamp.Create(createContract.EndDate.Value)
+            : null;
 
         var result = await service.Create(
-            createContract.GSRN,
+            [(createContract.GSRN, startDate, endDate)],
```

Comment:
Why take list as argument if we only call it with 1 object?

Reply:
This is endpoint is going to be deprecated in newer versions, so we only need 1 Create method when making a list